website/docs/software/overview.md

```diff
@@ -4,3 +4,22 @@ title: Overview
 sidebar_label: Overview
 slug: /software
 ---
+
+The following applications are available to support your Meshtastic network:
+  * The [firmware](/docs/software/device/device-software) to run on the devices
+  * Connect to the devices with our [Android app](/docs/software/android/android-installation)
+  * An [iOS application](/docs/software/ios/ios-development) is in the works
+  * [Meshtastic.js](/docs/software/js/getting-started) provides a javascript library to interface with devices
+  * [Meshtastic-python](/docs/software/python/python-installation) provides access from desktop computers
```

Comment:
Should we mention the CLI is bundled in this package? Or add another bullet point for it and point it appropriately to something within software/meshtastic-python/cli (if that exists?).

Reply:
Good point, added.

Reply:
Looks great!